3_navigation.ipynb

```diff
@@ -261,25 +526,53 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Advanced task</b>\n",
+    "    <p>Confirm what happens when we set a goal on the obstacle.</p>\n",
+    "    <p>Confirm what happens when we move a person in front of the robot while the robot is moving.</p>\n",
```

Comment:
"    <p>Check what happens if we move a person in front of the robot while the robot is moving.</p>\n",